src/ripple/nodestore/ShardSizeTuning.md

```diff
@@ -0,0 +1,212 @@
+# Shard Size Tuning
+
+The purpose of this document is to compare the sizes of shards containing
+varying amounts of ledgers.
+
+## Methodology
+
+One can see visually from a block explorer that typical mainnet ledger
+consists of about 30 offer transactions issued by about 8 different accounts,
+and several transactions of other types. To simulate this situation and
+similar situations we had constructed deterministic shards of differenet sizes,
+with different number of offer transactions per ledger and different number
+of accounts issuing these offers.
+
+Number of ledgers per shard starts from 256 to 16K with doubling the size on
+each step. We considered the following numbers of offers per ledger:
+0, 1, 5, 10 and 30. Also we considered case of 1 and 8 accounts issuing offers.
+For each constructed deterministic shard we counted its size. Finally we
+compared doubled size of the shard with N ledgers and the size of a shard with
+2*N ledgers where othere parameters such as number of offers and accounts are
+the same. This comparison is sufficient to determine which number or ledgers
+per shard leads to less storage size on the disk. 
+
+Note that we minimize total storage size on the disk, but not the size of each
+shard because data below shows that size of typical shard is not large than
```

Comment:
"shows that **the** size of a typical shard is not **larger**"

Reply:
Done.